scripts/globals/mobskills/Immortal_mind.lua

```diff
@@ -0,0 +1,43 @@
+-----------------------------------
+-- Immortal Mind
+-- Description: Gains Magic Attack Bonus and Magic Defense Bonus.
+-- Type: Self
+-- Range: Self
+-- Notes:
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
+require("scripts/globals/status")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local typeEffect1 = xi.effect.MAGIC_ATK_BOOST
+    local typeEffect2 = xi.effect.MAGIC_DEF_BOOST
+    local mabTotal = mob:getStatusEffect(xi.effect.MAGIC_ATK_BOOST)
+    local mdbTotal = mob:getStatusEffect(xi.effect.MAGIC_DEF_BOOST)
+
+    if (mob:getStatusEffect(xi.effect.MAGIC_ATK_BOOST) ~= nil) then -- mag atk bonus stacking
+        mabTotal = mabTotal:getPower() + 10
+    else
+        mabTotal = 10
+    end
```

Comment:
Why not...
```
local mabTotal = 10
if mob:getStatusEffect(xi.effect.MAGIC_ATK_BOOST) ~= nil then
    mabTotal = mabTotal:getPower() + 10
end
```

Also, uneeded parenthesis

Reply:
that get status already in a variable above. use that instead of executing another get

Reply:
mabTotal was defined as the effect itself, not its power. your are effectively saying effectID+10 instead of effect power +10

Reply:
`getStatusEffect()` gets the effect itself.. you want to get the power from that effect using `effect:getPower()` where effect is the effect entity, not its ID (so chain get status into get power)

Reply:
pseudo code, do not copypaste:
```lua
local effectID = xi.effect.dunnoLol
local effect = GetStatusEffect(effectID)
local effectPower = effect:getPower()
local powerNow - 0
if effect then
    powernow = effectpower + 10
end
```

Reply:
pseudo code, do not copypaste:
```lua
local effectID = xi.effect.dunnoLol
local effect = mob:getStatusEffect(effectID)
local effectPower = effect:getPower()
local powerNow - 0
if effect then
    powernow = effectpower + 10
end
```